project/MultiScalaProject.scala

```diff
@@ -172,34 +172,27 @@ object MultiScalaProject {
       case _        => id + major.replace('.', '_')
     }
 
-  def apply(id: String): MultiScalaProject =
-    apply(id, id, file(id), Nil)
-
-  def apply(id: String, base: File): MultiScalaProject =
-    apply(id, id, base, Nil)
-
-  def apply(
-      id: String,
-      name: String,
-      base: File
-  ): MultiScalaProject = apply(id, name, base, Nil)
-
-  def apply(
-      id: String,
-      base: File,
-      additionalIDEScalaVersions: List[String]
-  ): MultiScalaProject =
-    apply(id, id, base, additionalIDEScalaVersions)
-
   /** @param additionalIDEScalaVersions
    *    Allowed values: 3, 3-next, 2.13, 2.12.
    */
   def apply(
-      id: String,
       name: String,
-      base: File,
-      additionalIDEScalaVersions: List[String]
+      base: File = null,
+      additionalIDEScalaVersions: List[String] = Nil,
+      platform: String = null,
```

Comment:
This one seems to be a good candidate for enum  e.g.`JVM`, `Native` and `Default` for variant which is not cross compiled

Reply:
This shoudl also allow to simplify logic a bit in the body of this function

Reply:
done